recipe/meta.yaml

```diff
@@ -1,35 +1,89 @@
-{% set name = "openfst" %}
 {% set version = "1.8.2" %}
 
 package:
-  name: {{ name|lower }}
+  name: openfst
   version: {{ version }}
 
 source:
-  url: http://www.openfst.org/twiki/pub/FST/FstDownload/{{ name }}-{{ version }}.tar.gz
+  url: http://www.openfst.org/twiki/pub/FST/FstDownload/openfst-{{ version }}.tar.gz
   sha256: de987bf3624721c5d5ba321af95751898e4f4bb41c8a36e2d64f0627656d8b42
   patches:
-    - osx_arm64_configure.patch  # [build_platform != target_platform]
+    - patches/0001-Remove-float-equality-check-in-cross-compiling.patch  # [build_platform != target_platform]
+    - patches/0002-add-missing-header.patch
+    - patches/0003-add-no-undefined-to-some-LDFLAGS.patch                # [win]
+    - patches/0004-Fix-file-closing-on-windows.patch                # [win]
 
 build:
   number: 1
-  skip: true  # [win]
+  run_exports:
+  # seems to be changing even with patch releases
+  # https://abi-laboratory.pro/tracker/timeline/openfst/
+    - {{ pin_subpackage('openfst', max_pin='x.x.x') }}
 
 requirements:
   build:
     - {{ compiler('cxx') }}
-    - make
-    - gnuconfig   # [unix]
+    - make                   # [unix]
+    - gnuconfig              # [unix]
+    - cmake  # [win]
+    - ninja  # [win]
+    - m2-patch  # [win]
+  host:
+    - dlfcn-win32            # [win]
+    - icu
```

Comment:
Was this an oversight previously that also Unix had no icu?

Reply:
OpenFst doesn't use libicu. There is a header called `<fst/icu.h>` but it just implements the pieces that (in an early version) were borrowed from libicu.